src/_lib/sass/src/_variables.scss

```diff
@@ -9,11 +9,18 @@ $secondary-base: #3c8f9a !default;
 $tertiary-base: #ff3d64 !default;
 $alt-base: #ffcc33 !default;
 $coolgray-base: #333c4e !default;
+$levels-base: #121417 !default;
+
+$levels-color-settings: (
+    "name": "levels",
+    "color": $levels-base,
+    "light-steps": (3, 6, 9, 12, 15, 18),
+    "dark-steps": (-8)
+);
```

Comment:
_This comment applies to the level0-N variable declarations below as well_

I know we talked about this before but I don't remember what we said! Now that I see it, though, I would say that rolling this into the current colours implementation is unnecessary. There's a couple of reasons why I say that. 

1. We don't have any dark steps but I assume we're providing one here because it breaks if we don't
2. We don't want users to access this via get-color

With that in mind I would declare $levels-base as you already have and declare the other levels by directly using the [sass lighten](https://sass-lang.com/documentation/modules/color#lighten) function.

e.g. 

$level0: $levels-base;
$level1: lighten($levels-base, 3);
....
....
$level5: lighten($levels-base, 18);

Reply:
@garethdn Can't reply to the generic comment, so will address here:
_1. The active state on list items looks a little strange with border-radius_ - I'll fix the wrong radius on the first & last items, but Colm's asking whether you know if we use the blue left border anywhere? Sidebar uses its own styles set in Spike. 
_2. The email input component has a "whiter" input when you click into it_ - fixed
_3. The modal header is still a gray color (not sure if intentional)_ - I haven't done modals yet.

Reply:
> Colm's asking whether you know if we use the blue left border anywhere? Sidebar uses its own styles set in Spike.

I forgot that we basically completely override the styles in the sidebar. In that case I'd say the only place where this active style is used in Spike is probably in discover management